coralproject/cay
# Tests for actions and reducers

## What this PR does:

Implements tests for tag actions and reducers
## To test this PR:
- `npm install`
- Run `npm test`
- Tests should exist for TagActions and TagReducer
- Tests should pass
## Notes:
- In order to prevent problems with testing, internationalization has been stubbed.
- Actions are being tested with [redux-mock-store](https://www.npmjs.com/package/redux-mock-store). This catalogs the actions that the store has received so that they can be verified.
- Actions which interact with the server are being mocked via [fetch-mock](https://www.npmjs.com/package/fetch-mock). This lets you mock requests from an external server for testing.
